C-Plus-Plus/cp/Maximum_In_Array.cpp

```diff
@@ -31,12 +31,13 @@
  * @source: https://practice.geeksforgeeks.org/problems/maximum-in-struct-array/1
 */
 
-#include <iostream>
+#include <bits/stdc++.h>
+using namespace std;
 #include <vector>
```

Comment:
As you have added <bits/stdc++.h> you don't need to add any other header. Remove this line.

Reply:
yes, i have corrected it
Thanks for mentioning!